db/tweets.sql

```diff
@@ -0,0 +1,42 @@
+-- USERDATA table for holding the user information (USERNAME, user full name, and user ID)
```

Comment:
Can you make the parameters be more consistent even in your comments? You would probably want to put exactly the same variable names as the column names which you defined in the table.